.github/workflows/tag.yml

```diff
@@ -3,21 +3,25 @@ name: Try Runtime on New Tags
 on:
   push:
     tags:
-      - 'v[0-9]+.[0-9]+.[0-9]+'
+      - '*'
+      # - 'v[0-9]+.[0-9]+.[0-9]+'
```

Comment:
I think `- 'v[0-9]+.[0-9]+.[0-9]+'` is great. Just for testing the CI you use `*`, right?